Integrations/integration-Ipinfo.yml

```diff
@@ -50,7 +55,7 @@ script:
             }
             return reply;
         case 'ipinfo_field':
-            return sendRequest(base+args.ip+'/'+args.field, false);
+            return sendRequest(base+args.ip+'/'+args.field+'?token='+args.token, false);
```

Comment:
There is no `token` argument, should be `params.token` .
Please create a global `token` variable and use it in both URL addresses.